rocAL/rocAL/include/meta_data.h

```diff
@@ -181,7 +234,55 @@ struct BoundingBoxBatch: public MetaDataBatch
         return std::make_shared<BoundingBoxBatch>(*this);
     }
 };
+
+struct KeyPointBatch : public MetaDataBatch
+{
+    void clear() override
+    {
+        _img_sizes.clear();
+        _joints_data = {};
+        _bb_cords.clear();
+        _bb_label_ids.clear();
+    }
+    MetaDataBatch&  operator += (MetaDataBatch& other) override
+    {
+        _img_sizes.insert(_img_sizes.end(),other.get_img_sizes_batch().begin(), other.get_img_sizes_batch().end());
```

Comment:
If it is adding element to the end of the vector, please use push_back instead.

Reply:
Hi Rajy, we are not adding a single element here. We are adding multiple ImgSize objects present in _other_ instance of KeyPointBatch. This call is used in the += operator call of KeyPointBatch

We need to use a loop if we need to use push_back instead of insert